.github/workflows/publish-runtime.yml

```diff
@@ -17,6 +17,8 @@ on:
         default: tanssi-only
         options:
           - tanssi-only
+          - flashbox-only
+          - dancebox-only
           - templates-only
```

Comment:
Maybe also add options for releasing one template instead of both?

Reply:
added